src/components/inversionSolution/InversionSolutionHazardTab.tsx

```diff
@@ -53,30 +88,53 @@ const InversionSolutionHazardTab: React.FC<InversionSolutionHazardTabProps> = ({
           </Typography>
           <Box style={{ width: '100%', padding: '1rem' }}>
             <XYChart
-              height={700}
-              width={1200}
+              height={yMax}
+              width={xMax}
               xScale={{ type: 'log', domain: [1e-3, 10] }}
               yScale={{ type: 'log', domain: [1e-13, 2.0] }}
             >
               <AnimatedAxis orientation="bottom" label="Ground Motion (g)" />
               <AnimatedAxis orientation="left" label="Annual Frequency of Exceedance" />
-              <AnimatedLineSeries
-                dataKey="hazard plot"
-                data={filteredData}
-                xAccessor={(d) => d.x}
-                yAccessor={(d) => d.y}
-              />
-              <Grid rows={true} columns={true} />
+              {Object.keys(filteredData).map((key, index) => {
+                return (
+                  <AnimatedLineSeries
+                    key={key}
+                    dataKey={key}
+                    data={filteredData[key]}
+                    xAccessor={(d: XY) => d?.x}
+                    yAccessor={(d: XY) => d?.y}
+                    stroke={colors[index]}
+                  />
+                );
+              })}
+              <Grid rows={true} columns={true} lineStyle={{ opacity: '90%' }} />
```

Comment:
the ={true} is not needed here, React shorthand :)

Reply:
thanks!